imap/reconstruct.c

```diff
@@ -152,16 +152,21 @@ int main(int argc, char **argv)
     struct buf buf = BUF_INITIALIZER;
     char *alt_config = NULL;
     char *start_part = NULL;
+    char *path = NULL;
```

Comment:
nit: I might call this `header_path` or something, so that someone reading the reconstruct source instead of the man page can make a good guess what kind of value they need to supply.  I'll probably call this `--header-path` when I do the long options conversion later.